async/kafka/kafka.go

```diff
@@ -112,14 +110,18 @@ func (f *Factory) Create() (async.Consumer, error) {
 	c := &consumer{
 		brokers:     f.brokers,
 		topic:       f.topic,
-		group:       f.group,
 		traceTag:    opentracing.Tag{Key: "group", Value: f.group},
 		cfg:         config,
 		contentType: f.ct,
 		buffer:      0,
```

Comment:
The buffer here is a tricky thing.
0 is good for consumer group but bad for the normal consumer... we should provide different defaults for each case